pkg/report/sarif.go

```diff
@@ -348,6 +350,14 @@ func clearURI(s string) string {
 	return strings.ReplaceAll(strings.ReplaceAll(s, "\\", "/"), "git::https:/", "")
 }
 
+func toUri(str string) *url.URL {
+	uri, err := url.Parse(str)
+	if err != nil {
+		log.Error("Error while parsing URI", log.Err(err))
```

Comment:
let's add `sarif` prefix.
```suggestion
		logger := log.WithPrefix("sarif")
		logger.Error("Unable to parse URI", log.String("URI", str), log.Err(err))
```